test/functional/developer_portal/admin/messages/outbox_controller_test.rb

```diff
@@ -39,4 +39,18 @@ def test_index
     assert assigned_drop_variables.include?('messages')
     assert assigned_drop_variables.include?('pagination')
   end
+
+  test "Not Creates invalid message" do
+    buyer = FactoryBot.create :buyer_account, :provider_account => @provider
+    res = post :create, params: { message: { subject: nil, :body => "message with nil subject" }, :to => buyer.id }
```

Comment:
Here and on line 52 `res` variable is not used.

Also, I'm wondering if we need to run `MessageWorker.drain`.

Reply:
Yes we can remove res and yes we can run MessageWorker.drain, that woul drain and make test  more clear

Changes are done